src/test/scala/org/clulab/reach/TestFeedback2.scala

```diff
@@ -56,8 +56,13 @@ class TestFeedback2 extends FlatSpec with Matchers {
   }
 
   val s7 = "We observed that endogenous ASPP2 translocates from cell/cell junctions to the cytosol/nucleus following RAS activation"
-  s7 should "contain 1 or more translocation events" in {
-    // TODO: we should pick 4 translocation events here - ENRIQUE
+  s7 should "contain 2 translocation events" in {
+    // TODO: we should pick 2 translocation events here - ENRIQUE
+    // ENRIQUE: Cell is in the black list of the NER, so there will be only two events
```

Comment:
I agree with Enrique. Ok to change the test.

On Fri, Jul 22, 2016 at 1:31 AM, enoriega <notifications@github.com> wrote:

> In src/test/scala/org/clulab/reach/TestFeedback2.scala
> <https://github.com/clulab/reach/pull/273#discussion_r71799492>:
>
> > @@ -56,8 +56,13 @@ class TestFeedback2 extends FlatSpec with Matchers {
> >    }
> >
> >    val s7 = "We observed that endogenous ASPP2 translocates from cell/cell junctions to the cytosol/nucleus following RAS activation"
> > -  s7 should "contain 1 or more translocation events" in {
> > -    // TODO: we should pick 4 translocation events here - ENRIQUE
> > +  s7 should "contain 2 translocation events" in {
> > +    // TODO: we should pick 2 translocation events here - ENRIQUE
> > +    // ENRIQUE: Cell is in the black list of the NER, so there will be only two events
>
> It works correctly. There are two sources and two destinations, but one of
> the sources is cell which is filtered by the NER. So only two
> translocations match, where the source is cell junctions and the
> destinations are cytosol and nucleus
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/clulab/reach/pull/273/files/79aca9a19bb75a23e18634dcf74329728abe80f6#r71799492>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABH-zs1MOHyz62_uAnyiUTQHxogvUQzSks5qX_NIgaJpZM4JSP_d>
> .
>